packages/documentation-framework/package.json

```diff
@@ -60,7 +60,7 @@
     "style-to-object": "0.3.0",
     "to-vfile": "6.1.0",
     "typedoc": "0.23.0",
-    "typescript": "4.3.5",
+    "typescript": "4.7.4",
```

Comment:
This update looks fine to me.  However long term I wondering if most of the dependencies should be moved to dev and peer.  @wise-king-sullyman Do you see any issues with moving these dependencies in the future?

Reply:
Yeah I'm not sure how many should be moved, but it is probably something that should be investigated.